Documentation/packaging-installation.md

```diff
@@ -0,0 +1,82 @@
+# .NET Core Packaging and Installation
+
+This document helps you install or package a .NET Core SDK built using source-build.
+
+The SDK built by source-build is generally not portable. That means it will work on the same operating system where it was built. It will not work on older operating systems. It may work on newer operating systems.
+
+The built SDK is generally located at `artifacts/${ARCHITECTURE}/Release/dotnet-sdk-${SDK_VERSION}-${RUNTIME_ID}.tar.gz`.
+
+## Using the SDK directly (install per-user)
+
+To use the SDK directly, unpack the SDK to any directory and then use the `dotnet` executable from it.
+
+You can find more details on how to manually install an SDK from a tarball at [manually installing an SDK](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#manual-install). The same steps listed should work for all Linux distributions where `bash` is the default shell.
+
+## Installing the SDK globally
+
+If you want to install the SDK globally (and not per user), here are some suggestions.
+
+1. Extract the tarball to a distribution-appropriate location such as `/usr/local/lib64/dotnet/`, `/usr/local/lib/dotnet/`, or `/usr/local/lib/x86_64-linux-gnu/dotnet/`.
+
+2. Create a symlink from `/usr/local/bin/dotnet` (or equivalent) to where you installed the SDK in the previous step. This lets users simply run `dotnet` and have it work.
+
+3. Create an `/etc/dotnet/install_location` and add the location where you installed the SDK in there. This file is used by [.NET Core to find the SDK/Runtime location](https://github.com/dotnet/designs/blob/master/accepted/2020/install-locations.md).
+
+4. Define `DOTNET_ROOT` and update `PATH` by saving the following as `/etc/profile.d/dotnet-local.sh` (or equivalent)
+
+   ```bash
+   # Set location for AppHost lookup
+   [ -z "$DOTNET_ROOT" ] && export DOTNET_ROOT=/usr/local/lib64/dotnet
+
+   # Add dotnet tools directory to PATH
+   DOTNET_TOOLS_PATH="$HOME/.dotnet/tools"
+   case "$PATH" in
+       *"$DOTNET_TOOLS_PATH"* ) true ;;
+       * ) PATH="$PATH:$DOTNET_TOOLS_PATH" ;;
+   esac
+   ```
+
+   This allows apphost-lookup to work via `DOTNET_ROOT` and allows users to easily use dotnet tools directly after a `dotnet tool install`. This snippet only works on `bash`. You will need to adapt it for other shells.
```

Comment:
The snippet looks posixy to me and seems to work in an Ubuntu container with `dash`. On the other hand, `zsh` doesn't load `profile.d` and has some other way of doing things entirely. `will need to adapt it` => `may need to adapt it`, or am I missing something?

(I might not have tested `dash` properly, not sure if it might have some compat modes on.)

Reply:
I think you are right. We wrote this snippet in a way to be explicitly compatible with `sh` (unless we introduced some bugs).